### PR TITLE
Adding azure functions docs to the python and node integrations

### PR DESCRIPTION
**Pull request by ajjindal**

### CODE REVIEW

File: src/collections/_documentation/platforms/python/gcp_functions.md [hunk 1]
```diff
@@ -9,26 +9,6 @@ sentry_sdk
 ```
 
 You can use the GCP Functions integration for the Python SDK like this:
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/gcp_functions.md:11]: Can you use `<!-- WIZARD -->` like in the other pages?

File: src/collections/_documentation/platforms/python/azure_functions.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Azure Functions
+sidebar_order: 9
+---
+
+Create a .venv in the local machine on VS Code, create a Python function, and install the sentry SDK:
+```python
+sentry_sdk==0.14.4
```
Comment (untitaker) [inline src/collections/_documentation/platforms/python/azure_functions.md:8]: Please copy this from the sourcecode of other pages, hardcoding the version here is not good.

File: src/_data/platforms.yml [hunk 3]
```diff
@@ -52,6 +52,16 @@
       wizard_parent: node
       wizard:
         - _documentation/platforms/node/gcp_functions.md
+-
+      slug: nodeazurefunctions
+      support_level: production
+      type: framework
+      name: Azure Functions
+      doc_link: /platforms/python/azure_functions/
```
Comment (kamilogorek) [inline src/_data/platforms.yml:60]:
```suggestion
      doc_link: /platforms/node/azure_functions/
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Sentry.init({
+  dsn: "<Your dsn>",
+});
+
+module.exports = async function (context, req) {
+    try{
+        callUndefinedFunction();  // Call undefined function.
+    }catch(error){
+        Sentry.captureException(error);  // Capture the exception in Sentry dashboard.
+        Sentry.flush(2000);
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:24]:
```suggestion
        await Sentry.flush(2000);
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Sentry.init({
+  dsn: "<Your dsn>",
+});
+
+module.exports = async function (context, req) {
+    try{
+        callUndefinedFunction();  // Call undefined function.
+    }catch(error){
+        Sentry.captureException(error);  // Capture the exception in Sentry dashboard.
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:23]:
```suggestion
    try {
        callUndefinedFunction(); // Call undefined function.
    } catch (error) {
        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    context.res = {
+        status: 200, /* Defaults to 200 */
+        body: "Hello from Azure cloud function"
+    };
+}
+````
+
+You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
+
+Note: You need to call both `CaptureException` and `Flush` to report errors into Sentry.
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:36]:
```suggestion
Note: You need to call both `captureException` and `flush` to report errors into Sentry.
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+title: Azure Functions
+sidebar_order: 9
+---
+
+Install our Nodejs SDK in Azure:
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:6]:
```suggestion
Install our Node.js SDK in Azure:
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+```basic
+ npm install @sentry/node
+```
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:9]:
```suggestion
```bash
npm install @sentry/node
```
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
+        await Sentry.flush(2000);
+    }
+
+    context.res = {
+        status: 200, /* Defaults to 200 */
+        body: "Hello from Azure cloud function"
+    };
+}
+```
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:32]:
```suggestion
```js
const Sentry = require("@sentry/node");

Sentry.init({
    dsn: "___PUBLIC_DSN___"
});

module.exports = async function (context, req) {
    try {
        await callUndefinedFunction(); // Call undefined function.
    } catch (error) {
        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
        await Sentry.flush(2000);
    }

    context.res = {
        status: 200, /* Defaults to 200 */
        body: "Hello from Azure cloud function"
    };
}
```
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
+
+Note: You need to call both `captureException` and `flush` to report errors into Sentry.
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/azure_functions.md:36]:
```suggestion
Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
```

File: src/collections/_documentation/platforms/node/gcp_functions.md [hunk 11]
```diff
@@ -34,7 +34,7 @@ exports.cloud_handler = (event, context) => {
 ```
 You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
 
-Note: You need to call both `CaptureException` and `Flush` to report errors into Sentry.
+Note: You need to call both `captureException` and `flush` to report errors into Sentry.
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/gcp_functions.md:37]:
```suggestion
Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
```

**Pull request merged**